docs/Using Price Feeds/feed-registry.html

```diff
@@ -118,376 +127,4 @@
 
```

Comment:
General remark:
You should have only one <_h1_> (_#_ markdown) in your page. the <_h1_> is already taken by the title of the page. Hence, you should start with _##_ then _###_ ...